packages/js/product-editor/src/icons/gift.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+import { SVG, Path, Rect } from '@wordpress/components';
+
+const gift = (
+	<SVG viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<Rect
+			x="-0.75"
+			y="-0.75"
+			fill="none"
+			width="9.5"
+			height="14.5"
+			transform="matrix(3.97376e-08 -1 -1 -4.80825e-08 18.5 18.5)"
+			stroke="#1E1E1E"
```

Comment:
Hardcoding the color doesn't feel great. It seems that all icons do that, right?